src/managed/CommonManaged.props

```diff
@@ -11,6 +11,8 @@
     <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
     <IncludeSymbols>true</IncludeSymbols>
     <Serviceable>true</Serviceable>
+    <PackageLicenseUrl>https://github.com/dotnet/core-setup/blob/master/LICENSE.TXT</PackageLicenseUrl>
```

Comment:
Then, should we also align this? 

https://github.com/dotnet/core-setup/blob/master/src/pkg/dir.props#L19